### PR TITLE
https://unit.atlassian.net/browse/NP-44611

### PR DESCRIPTION
**Pull request by StigNorland**

### CODE REVIEW

File: template.yaml [hunk 1]
```diff
@@ -136,29 +136,30 @@ Resources:
...
+          schemas:
+            Problem:
+              type: object
+              properties:
+                title:
+                  type: string
+                status:
+                  type: integer
+                detail:
+                  type: string
```
[the remark this answers is not in the document]
Comment (sondrev) [reply template.yaml:162]: Its not equal yo your other PRs

[the remark this answers is not in the document]
Comment (StigNorland) [reply template.yaml:162]: oisann!

File: template.yaml [hunk 2]
```diff
@@ -48,7 +48,7 @@ Resources:
       StageName: Prod
       EndpointConfiguration: REGIONAL
       DefinitionBody:
-        swagger: '2.0'
+        swagger: '3.0'
```
Comment (sondrev) [inline template.yaml:51]: openapi: 3.0.3 ?

**Review by sondrev — CHANGES_REQUESTED** [review]

File: template.yaml (hunk 2 above)
Comment (StigNorland) [reply template.yaml:51]: Lager ny subtask, ekstraher til egen openapi fil

**Pull request merged**